src/polyfills.ts

```diff
@@ -1,7 +1,12 @@
 import 'polyfill-object.fromentries'
 
+import { ResizeObserver } from '@juggle/resize-observer'
```

Comment:
Why change our dependency instead of just importing the existing package in `src/polyfills`? What makes `@juggle/resize-observer` preferable to `resize-observer-polyfill`?

NB: `resize-observer-polyfill` has much more usage, although it is larger and staler.

Reply:
it just looked like it was more maintained tbh, but we can leave the old one if better

Reply:
if you look at yarn.lock we already had `@juggle/resize-observer` somewhere - the net of this is just removing a dep